piker/data/marketstore.py

```diff
@@ -642,30 +649,30 @@ async def tsdb_history_update(
     ):
         profiler(f'opened feed for {fqsn}')
 
-        to_append = feed.shm.array
-        to_prepend = None
+        # to_append = feed.hist_shm.array
+        # to_prepend = None
 
         if fqsn:
             symbol = feed.symbols.get(fqsn)
             if symbol:
                 fqsn = symbol.front_fqsn()
 
             # diff db history with shm and only write the missing portions
-            ohlcv = feed.shm.array
+            # ohlcv = feed.hist_shm.array
 
             # TODO: use pg profiler
-            tsdb_arrays = await storage.read_ohlcv(fqsn)
-            # hist diffing
-            if tsdb_arrays:
-                for secs in (1, 60):
-                    ts = tsdb_arrays.get(secs)
-                    if ts is not None and len(ts):
-                        # these aren't currently used but can be referenced from
-                        # within the embedded ipython shell below.
-                        to_append = ohlcv[ohlcv['time'] > ts['Epoch'][-1]]
-                        to_prepend = ohlcv[ohlcv['time'] < ts['Epoch'][0]]
-
-            profiler('Finished db arrays diffs')
+            # for secs in (1, 60):
```

Comment:
Yeah just ignore most of this stuff since it's part of provisional tools we likely want for interactive tsdb mucking.

It relies on https://github.com/goodboy/tractor/pull/306 which is still far from ready 😂